homeassistant/components/zwave/__init__.py

```diff
@@ -310,6 +317,17 @@ def log_all(signal, value=None):
 
         dispatcher.connect(log_all, weak=False)
 
+    @websocket_api.async_response
```

Comment:
Why async, doesn't seem async?

Reply:
I'm still not completely up to speed on the difference between when to use sync/async methods/functions for different things, especially with regards to websocket responses.

Reply:
if we need to do I/O or run a sync method, it should be async. If we can just read the data from memory, no need for async.